### DISCUSSION TITLE
external questions

### DISCUSSION THREAD

**Discussion by badytest** [imartinez/privateGPT#243] [General]
privateGPT can't answer external questions, I mean out of source documents info. right?
I think it is finetuned with source documents.
Now I want to use the main models which know a lot of general information without finetune. can I do that?

**Comment by thekit** [comment]
privateGPT can answer general knowledge questions using the model it started with. I am on vicuna and here is a question and answer I just got. note the mistake where it references the state of the union because that is the only document it knows about regardless of whether is is relevant to the general knowledge question. the documents that trained the vicuna model are not indexed by privateGPT so we don't know where it is drawing its general knowledge from

> Question:
what kind of mammal is a vole?

> Answer:
 A vole is a type of rodent that is native to South America. It is also known as a mara, mara de la Sierra, or mara de la sierra.

> source_documents\state_of_the_union.txt:
Please rise if you are able and show that, Yes, we the United States of America stand with the Ukrainian people.

Here is how I got the vicuna model to work:
https://github.com/imartinez/privateGPT/discussions/233

however if you don't care about fine tuning on your local documents, try installing GPT4All using their official installer packages from their main website. it comes with a nice app ux and a file manager to download and manage your various models. privateGPT is catching up to add these features, but the pull request has not been merged yet.

https://gpt4all.io/index.html
